includes/Rest/Site.php

```diff
@@ -11,6 +11,7 @@ class Site extends RestController {
     public function register_routes() {
         $this->post( '/settings', 'save_settings', 'can_manage_settings' );
         $this->post( '/settings/save-options', 'save_options', 'manage_options' );
+        $this->delete( '', 'delete_site' );
```

Comment:
Missing permission callback. We can not allow anyone can delete any sites information without proper authorization.